my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerable'
+
+class MyList
+  def initialize(*elements)
+    @list = elements
+  end
+
+  include MyEnumerable
+
+  # Implement each method
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+puts(list.all? { |e| e < 5 }) # Output: true
+puts(list.all? { |e| e > 5 }) # Output: false
+puts(list.any? { |e| e == 2 }) # Output: true
+puts(list.any? { |e| e == 5 }) # Output: false
+puts(list.filter(&:even?)) # Output: [2, 4]
```

Comment:
- [ ] As you mentioned here and as mentioned in the project requirements this should return `[2, 4]` but for you, the result is a little bit different, please kindly fix that 🙏🏾 

<img width="537" alt="Screenshot 2023-06-15 at 20 42 35" src="https://github.com/Metaverse-Nabeel/MyEnumerables/assets/74501523/ff464786-e4de-45b1-bd3c-90b8457c9c1e">

- [ ] It would be better if @Metaverse-Nabeel add more contributions to the project as well 🙃 😁